curriculum/challenges/english/08-coding-interview-prep/rosetta-code/count-the-coins.english.md

```diff
@@ -37,7 +37,7 @@ Implement a function to determine how many ways there are to make change for a d
 tests:
   - text: <code>countCoins</code> should be a function.
     testString: assert(typeof countCoins === 'function');
-  - text: <code>countCoints()</code> should return 242.
```

Comment:
I try not to fix things which seem to be intentionally pointing to a spelling error, but this appears to just be a spelling error. (e.g. misspellings of `Roosevelt` and `function`)